docs/index.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta http-equiv="refresh" content="0; url=https://docs.revenuecat.com/android/5.1.0-SNAPSHOT/index.html" />
+</head>
+<body>
+</body>
+</html>
```

Comment:
this is a new file tho 😂  I just created it today so it would redirect automatically

Reply:
Oh. I had uploaded one to AWS so it would redirect to the `https://sdk.revenuecat.com/android/docs/index.html` if accessing `https://sdk.revenuecat.com/android/index.html`. But I think that's long time gone 😄